pkgs/top-level/aliases.nix

```diff
@@ -377,6 +377,7 @@ mapAliases ({
   fast-neural-doodle = throw "fast-neural-doodle has been removed, as the upstream project has been abandoned"; # Added 2020-03-28
   fastnlo = fastnlo_toolkit; # Added 2021-04-24
   fedora-coreos-config-transpiler = throw "fedora-coreos-config-transpiler has been renamed to 'butane'"; # Added 2021-04-13
+  feedreader = throw "FeedReader is no longer activily maintained since 2019. The developer is working on a spiritual successor called NewsFlash."; # Added 2020-05-03
```

Comment:
```suggestion
  feedreader = throw "FeedReader is no longer activily maintained since 2019. The developer is working on a spiritual successor called NewsFlash."; # Added 2022-05-03
```

Reply:
oh no what happened the last 2 years :grimacing: